lib/htmlValidator.js

```diff
@@ -0,0 +1,73 @@
+var colors = require('colors'),
+    validator = require('html-validator'),
+    path = require('path');
+
+colors.setTheme({
+  warn: 'yellow',
+  error: 'red'
+});
+
+module.exports = function(app) {
+  var params = app.get('params'),
+      options,
+      errorPage = path.normalize(__dirname + '/../defaultErrorPages/views/validatorErrors'),
+      validatorDisabled;
+
+  if (!params.enableValidator) {
+    validatorDisabled = true;
+  }
+
+  if (!params.htmlValidator) {
+    options = {
+      format: 'text'
+    };
+  }
+  else {
+    options = params.htmlValidator;
+  }
+
+  app.use(function(req, res, next) {
+
+    // get a reference to the original render
+    var renderReference = res.render;
+
+    // override it
+    res.render = function(view, model, fn) {
+
+      var thisReference = this;
+
+      // get the html for this specific render
+      app.render(view, model, function(err, html) {
+
+        if (process.env.NODE_ENV === 'development' && !validatorDisabled) {
+          // run HTML validator and display errors on page
+          options.data = html;
+
+          validator(options, function(error, htmlErrorData) {
+            var model = {};
```

Comment:
you probably should've tested this. 😛 

this line breaks the whole damn thing.

`model` is provided by the render method. you don't need to destroy the whole frigging model by redefining it for no frigging reason